pyproject.toml

```diff
@@ -134,6 +137,8 @@ filterwarnings = [
     "ignore:The unit 'Angstrom' has been deprecated in the VOUnit standard\\. Suggested.* 0\\.1nm\\.",
     "ignore:((.|\n)*)Sentinel is not a public part of the traitlets API((.|\n)*)",
     "ignore:datetime\\.datetime\\.utcfromtimestamp:DeprecationWarning",  # asdf + dateutil<=2.8.2 + Python 3.12
+    "ignore:'audioop' is deprecated and slated for removal in Python 3.13",
```

Comment:
do we need a follow-up to address this before 3.13 comes out?  What should be used instead?

Reply:
This is an upstream issue, I can make a note of these warnings and pass them along to the Strauss team.

Reply:
Follow-up for tracking that information #3332

Reply:
Please link issue to upstream repo here, thx.